src/components/Cart/Cart.js

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import axios from "axios";
+import { useDataContext } from "../../context/data-context";
+import "./cart.css";
+import CartCard from "./CartCard";
+import { useEffect } from "react";
+import PriceCard from "./PriceCard";
+
+const Cart = () => {
+  const { state, dispatch } = useDataContext();
+  const localToken = localStorage.getItem("token");
+
+  const getCartProduct = async (localToken) => {
+    const response = await axios.get("/api/user/cart", {
+      headers: { authorization: localToken },
+    });
+
+    dispatch({
+      type: "GET_CART",
+      payload: response.data.cart,
+    });
+  };
+
+  const removeFromCart = async (_id) => {
+    const localToken = localStorage.getItem("token");
+    const response = await axios.delete(`/api/user/cart/${_id}`, {
+      headers: {
+        authorization: localToken,
+      },
+    });
+    if (response.status === 200) {
+      dispatch({ type: "REMOVE_FROM_CART", payload: response.data.cart });
+    } else {
+      console.warn(" error ", response);
+    }
```

Comment:
A suggestion here is that you could separate out your utility functions in your utility folder and the utility files should be named `name.util.js` and network calls in the services folder and the service file should be named `name.service.js`.

Reply:
yes I will do these refactor in separate pr